lib/oli_web/router.ex

```diff
@@ -789,9 +796,8 @@ defmodule OliWeb.Router do
   scope "/sections/workspace/", OliWeb do
```

Comment:
I just realized this routing is not quite right. This scope should not include 'sections', Instead it should simply be `/workspaces`. We can address this in a separate ticket